AnalyticsPlugin/iOS/README.md

```diff
@@ -0,0 +1,142 @@
+# ZappAnalyticsPluginExample-iOS
+
+The ZappAnalyticsPluginExample-iOS is an example project for creating a analytics provider plugin for the Applicaster Zapp Platform. You can use this example project as a reference for how to build your own analytics provider plugin.
+
+If you are not familiar with Zapp please visit [our website](http://applicaster.com/?page=product) for more details.
+
+The full [Zapp](http://zapp.applicaster.com) plugins documentation is available [here](http://zapp-tech-book.herokuapp.com).
```

Comment:
let's change the url to https://developer-zapp.applicaster.com (replace the herokuapp)